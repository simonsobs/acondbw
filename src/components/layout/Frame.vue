<template>
  <div>
    <v-navigation-drawer v-model="drawer" app clipped>
      <navigation></navigation>
    </v-navigation-drawer>
    <app-bar v-on:drawer="drawer = !drawer"></app-bar>
  </div>
</template>

<script>
import AppBar from "./AppBar";
import Navigation from "./Navigation";

export default {
  name: "Frame",
  components: {
    AppBar,
    Navigation,
  },
  data: () => ({
    drawer: true,
  }),
};
</script>
