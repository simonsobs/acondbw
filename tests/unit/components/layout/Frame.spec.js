import Vue from "vue";
import Vuetify from "vuetify";
import { mount, shallowMount, createLocalVue } from "@vue/test-utils";

import Frame from "@/components/layout/Frame";

Vue.use(Vuetify);

describe("Frame.vue", () => {
  let localVue;
  let vuetify;
  let wrapper;

  beforeEach(() => {
    localVue = createLocalVue();
    vuetify = new Vuetify();
    wrapper = shallowMount(Frame, {
      localVue,
      vuetify,
    });
  });

  it("match snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
