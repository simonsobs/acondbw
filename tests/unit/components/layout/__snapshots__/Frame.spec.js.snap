// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Frame.vue match snapshot 1`] = `
<div>
  <v-navigation-drawer-stub app="true" mobilebreakpoint="1264" clipped="true" height="100vh" minivariantwidth="56" src="" tag="nav" width="256" value="true">
    <navigation-stub></navigation-stub>
  </v-navigation-drawer-stub>
  <app-bar-stub></app-bar-stub>
</div>
`;
